modules/understanding-dns-operator.adoc

```diff
@@ -0,0 +1,21 @@
+// Module filename: nw-creating-secured-routes.adoc
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+[id="understanding-dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator deploys and manages CoreDNS to provide a name resolution
+service to pods that enables DNS-based Kubernetes Service discovery in
```

Comment:
This sentence is a bit awkward. Sounds at first like the pods enable DNS-based Kubernetes discovery...
Maybe s/that enables/enabling ??